problem_builder/tests/integration/test_data_export.py

```diff
@@ -82,3 +81,11 @@ def test_data_export(self):
         self.wait_until_visible(download_button)
         self.wait_until_visible(delete_button)
         self.assertIn('A report is available for download.', status_area.text)
+
+    def test_non_staff_disabled(self):
+        data_export = self.go_to_view()
+        self.assertRaises(NoSuchElementException, data_export.find_element_by_class_name, 'data-export-start')
```

Comment:
Nope, feel free to leave it. I don't think that's the usual way to code it, but I don't have any objections :)